### PR TITLE
[DRAFT] Add ComputeSharp animations to page transitions and the ContentDialog page

### PR DESCRIPTION
**Pull request by getrou**
- Add ComputeSharp as a nuget dependency
  - Bump WinAppSDK version to 1.6
- Add Shaders folder
  - Add RippleFade and TwirlDismiss shader effects
  - Add PixelShaderRenderer, ShaderPanel, and OverlayPanel to enable shaders and overlays in WinUI
  - Add CaptureHelper to facilitate taking captures of UI

### CODE REVIEW

File: nuget.config [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
     <!-- a local directory to allow testing nupkg files without pushing to a remote feed -->
     <add key="packagestore" value="packagestore" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (getrou) [inline nuget.config:15]: Going to need someone with permissions to add the ComputeSharp nuget packages to an approved feed.

File: WinUIGallery/WinUIGallery.csproj [hunk 2]
```diff
@@ -115,14 +118,14 @@
     </ItemGroup>
 
     <ItemGroup Condition="'$(IsInWinUIRepo)' != 'true'">
-        <PackageReference Update="Microsoft.WindowsAppSDK" Version="$(WindowsAppSdkPackageVersion)" />
+        <PackageReference Update="Microsoft.WindowsAppSDK" Version="1.6.240923002" />
```
Comment (ionvasilian) [inline WinUIGallery/WinUIGallery.csproj:121]: Why do we need the bump to 1.6?

Comment (getrou) [reply WinUIGallery/WinUIGallery.csproj:121]: ComputeSharp has a dependency on 1.6

File: WinUIGallery/Helper/WindowHelper.cs [hunk 3]
```diff
@@ -31,7 +31,7 @@ static public Window CreateWindow()
         {
             Window newWindow = new Window
             {
-                SystemBackdrop = new MicaBackdrop()
+                //SystemBackdrop = new MicaBackdrop()
```
Comment (jeffstall) [inline WinUIGallery/Helper/WindowHelper.cs:34]: Why?

Comment (getrou) [reply WinUIGallery/Helper/WindowHelper.cs:34]: The capture API doesn't capture Mica, so there was a visual glitch when it swapped in the capture to run the effect.

**Pull request merged**